chainercv/links/ssd/ssd.py

```diff
@@ -0,0 +1,257 @@
+from __future__ import division
+
+import itertools
+import numpy as np
+import six
+
+import chainer
+import chainer.functions as F
+from chainer import initializers
+import chainer.links as L
+
+from chainercv import transforms
+
+
+class SSD(chainer.Chain):
+    """Base class of Single Shot Multibox Detector [1].
+
+    This is a base class of Single Shot Multibox Detector.
+    It requires a feature extraction method and a preprocessing method.
+
+    [1] Wei Liu, Dragomir Anguelov, Dumitru Erhan, Christian Szegedy,
+    Scott Reed, Cheng-Yang Fu, Alexander C. Berg.
+    SSD: Single Shot MultiBox Detector. ECCV 2016.
+
+    Args:
+        n_class (int): The number of classes.
```

Comment:
No, my code classifies bounding boxes into `n_class + 1` classes. I think this design is good. I will note the number does not contain `background`. Does it make sense?